src/hooks/useLogDownloader/index.ts

```diff
@@ -37,19 +39,31 @@ const useLogDownloader = (url: string) => {
       .then((response) => response.text() || "")
       .then((text) => {
         setData(processLogString(text));
+        sendEvent({
+          name: "Log Downloaded",
+          duration: Date.now() - timeStart,
+          type: logType,
+        });
       })
       .catch((err: Error) => {
         leaveBreadcrumb("useLogDownloader", { url, err }, "error");
         reportError(err).severe();
         setError(err.message);
       })
       .finally(() => {
+        leaveBreadcrumb(
+          "useLogDownloader",
+          { url, time: Date.now() - timeStart },
+          "request"
+        );
+
         setIsLoading(false);
       });
     return () => {
       // Cancel the request if the component unmounts
       abortController.abort();
     };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
`sendEvent` technically isn't stable so I wanted to avoid rerunning this effect since it performs an expensive network request. its used in a `useCallback` effect within `useKeyboardShortcut`